src/ebay-tourtip/README.md

```diff
@@ -0,0 +1,84 @@
+| Name              | Type     | Stateful | Required | Description                                                                                                                                                  |
+| ----------------- | -------- | -------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `pointer`         | String   | No       | No       | options are `top-left`, `top`, `top-right`, `right`, `right-bottom`, `right-top`, `bottom-left`, `bottom-right`, `bottom`, `left`, `left-bottom`, `left-top` |
+| `overlayStyle`    | Object   | No       | No       | Style object to customize default values for the overlay. It can be used all CSS properties like `top`, `left`, `bottom`, `right`.                           |
+| `initialExpanded` | Boolean  | No       | No       | Open the tooltip on the initial render, needs to be true for the case of Tourtips                                                                                                                       |
+| `a11yCloseText`   | String   | No       | Yes      | A11y text for close button and mask.                                                                                                                         |
+| `aria-label`      | String   | No       | Yes      | A descriptive label of what the tourtip button represents (e.g. "Important information")
+| `onExpand`        | Function | No       | No       | overlay has been expanded                                                                                                                                    |
+| `onCollapse`      | Function | No       | No       | overlay has been collapsed                                                                                                                                   |
+
+## Child components
+
+Name | Required | Description
+--- | --- | ---
+`EbayTourtipHost`    | Yes | The custom host-button (trigger) for the tourtip
+`EbayTourtipHeading` | No | The heading to be displayed in the tourtip
+`EbayTourtipContent` | Yes | The content to be displayed in the tourtip
+
+## EbayTourtipHost Attributes
+
+| Name              | Type     | Stateful | Required | Description                                                                                                                                                  |
+| ----------------- | -------- | -------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `aria-label`      | String   | No       | Yes      | A descriptive label of what the tourtip button represents (e.g. "Important information"). Will overwrite `ariaLabel` prop from the `EbayTourtip`
+| `forwardedRef`      | Object   | No       | Yes      | A reference to the host element to be focused when component collapses
```

Comment:
This is not part of the public API, it is used internally

Reply:
yes, please put only official props into readme